protocol-configuration/mainnetAddresses.ts

```diff
@@ -1855,6 +1855,11 @@ const MainnetAddresses: MainnetAddresses = {
     artifactName: 'RariGovernanceTokenUniswapDistributor',
     address: '0x1fa69a416bcf8572577d3949b742fbb0a9cd98c7',
     category: AddressCategory.Governance
+  },
+  rariTurboFusePool: {
+    artifactName: 'unknown',
+    address: '0x1d9EEE473CC1B3b6D316740F5677Ef36E8f0329e',
```

Comment:
This should be the cToken I believe: [0x081E7C60bCB8A2e7E43076a2988068c0a6e69e27](https://etherscan.io/address/0x081E7C60bCB8A2e7E43076a2988068c0a6e69e27)

Reply:
You're right, was hunting around for that. Thanks